packages/ra-ui-materialui/src/form/TabbedForm.js

```diff
@@ -72,121 +73,122 @@ export const getTabFullPath = (tab, index, baseUrl) =>
+                                    isValidElement(tab)
+                                        ? React.cloneElement(tab, {
+                                              intent: 'content',
+                                              resource,
+                                              record,
+                                              basePath,
+                                              hidden: !routeProps.match,
+                                          })
+                                        : null
+                                }
+                            </Route>
+                        )
                 )}
-                <Divider />
-                <CardContentInner>
-                    {/* All tabs are rendered (not only the one in focus), to allow validation
-                    on tabs not in focus. The tabs receive a `hidden` property, which they'll
-                    use to hide the tab using CSS if it's not the one in focus.
-                    See https://github.com/marmelab/react-admin/issues/1866 */}
-                    {Children.map(
-                        children,
-                        (tab, index) =>
-                            tab && (
-                                <Route
-                                    exact
-                                    path={getTabFullPath(tab, index, url)}
-                                >
-                                    {routeProps =>
-                                        isValidElement(tab)
-                                            ? React.cloneElement(tab, {
-                                                  intent: 'content',
-                                                  resource,
-                                                  record,
-                                                  basePath,
-                                                  hidden: !routeProps.match,
-                                                  /**
-                                                   * Force redraw when the tab becomes active
-                                                   *
-                                                   * This is because the fields, decorated by redux-form and connect,
-                                                   * aren't redrawn by default when the tab becomes active.
-                                                   * Unfortunately, some material-ui fields (like multiline TextField)
-                                                   * compute their size based on the scrollHeight of a dummy DOM element,
-                                                   * and scrollHeight is 0 in a hidden div. So they must be redrawn
-                                                   * once the tab becomes active.
-                                                   *
-                                                   * @ref https://github.com/marmelab/react-admin/issues/1956
```

Comment:
Can you confirm this issue doesn't come back?

Reply:
I tried it yes